tools/run_tests/run_tests.py

```diff
@@ -1097,6 +1094,12 @@ def runs_per_test_type(arg_str):
         help='Generates a JUnit-compatible XML report')
 argp.add_argument('--report_suite_name', default='tests', type=str,
         help='Test suite name to use in generated JUnit XML report')
+argp.add_argument('--quiet_success',
```

Comment:
Looks like this option only makes sense when n > 1, maybe we don't need to expose it as an arg?